src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticListener.cs

```diff
@@ -199,13 +199,13 @@ public virtual void Dispose()
         /// <summary>
         /// When a DiagnosticListener is created it is given a name.   Return this.
         /// </summary>
-        public string Name { get; private set; }
+        public string? Name { get; private set; }
 
         /// <summary>
         /// Return the name for the ToString() to aid in debugging.
         /// </summary>
         /// <returns></returns>
-        public override string ToString()
+        public override string? ToString()
         {
             return Name;
```

Comment:
@stephentoub should we instead return empty here? As returning null is not recommended?

Reply:
Yeah. Empty strings aren't recommended either, but they're fairly common despite that.  Probably better to add `?? string.Empty` here.

Reply:
Ok